### ISSUE TITLE
[BUG] Multi-level Sider open all items

### ISSUE THREAD

**Issue by valentincosson** [refinedev/refine#3505]
### Describe the bug

Context:
On a multi-level sider like described here : https://refine.dev/docs/advanced-tutorials/multi-level-menu/

When you click on items with children, you close it, and you click on another item without children the items clicked before will be opened again

### Steps To Reproduce

1. Click on an item that has at least a child (like the two top on my screenshot)
<img width="182" alt="Capture d’écran 2023-01-31 à 18 01 13" src="https://user-images.githubusercontent.com/7271734/215831100-259f2983-1d92-43ef-91cf-aa0efcac9ccd.png">

2. Close it
<img width="182" alt="Capture d’écran 2023-01-31 à 18 01 29" src="https://user-images.githubusercontent.com/7271734/215831161-957c713c-13e7-4c35-867b-46d51c534d72.png">

3. Click on an item that doesn't have any child --> This opens up everything
 <img width="182" alt="Capture d’écran 2023-01-31 à 18 02 17" src="https://user-images.githubusercontent.com/7271734/215831352-67921a29-3ac3-4bf8-b71c-d0a08d71a676.png">

### Expected behavior

Doesn't open unwanted items.

### Desktop

Chrome / Firefox

### Additional Context

in case it's useful, the resource prop on my Refine Component :
I tried with differents "parentName" but it doesn't change

```
resources={[
              {
                name: "Clients",
                icon: <GroupsIcon />,
              },
              {
                name: "clients",
                parentName: "Clients",
                icon: <PersonIcon />,
                list: ClientsList,
                create: ClientCreate,
                edit: ClientEdit,
              },
              {
                name: "client-groups",
                parentName: "Clients",
                icon: <GroupsIcon />,
                list: ClientGroupsList,
                create: ClientGroupCreate,
                edit: ClientGroupEdit,
              },
              {
                name: "contracts",
                parentName: "Clients",
                icon: <ArticleIcon />,
                list: ContractsList,
                create: ContractCreate,
                edit: ContractEdit,
              },
              {
                name: "Vehicles",
                icon: <LocalShippingIcon />,
              },
              {
                name: "vehicles",
                parentName: "Vehicles",
                icon: <LocalShippingIcon />,
                list: VehiclesList,
                create: VehicleCreate,
                edit: VehicleEdit,
              },
              {
                name: "semi-trucks",
                parentName: "Vehicles",
                icon: <LinkIcon />,
                list: SemiTrucksList,
                create: SemiTrucksCreate,
                edit: SemiTrucksEdit,
              },
              {
                name: "drivers",
                icon: <AirlineSeatReclineNormalIcon />,
                list: DriversList,
                create: DriverCreate,
                edit: DriverEdit,
              },
              {
                name: "delivery-rounds",
                icon: <TodayIcon />,
                list: DeliveryRoundsList,
              },
            ]}

**Comment by yildirayunlu** [comment]
Hey @valentincosson 

We are sorry for this issue. Which UI library are you using (antd,mui,mantine,chakra)?

**Comment by valentincosson** [comment]
Hello @yildirayunlu !
I use MUI. Do you need more informations ?

**Comment by yildirayunlu** [comment]
> 

Thank you for information.

**Comment by yildirayunlu** [comment]
> Hello @yildirayunlu ! I use MUI. Do you need more informations ?

Thank you for information 🙏🏼

**Comment by aliemir** [comment]
Hey @valentincosson, we've released `@pankod/refine-mui@3.61.5` which includes the fix for this issue. 🚀  Please upgrade your `@pankod/refine-mui` dependency to `latest`. Let us know if the issue continues 🙏
